datasets/za/pmg_legislators/za_pmg_legislators.yaml

```diff
@@ -0,0 +1,35 @@
+title: South Africa National and Provincial Legislature and Executives
+entry_point: crawler.py
+prefix: za-pmg-leg
+coverage:
+  frequency: daily
+  start: 2024-08-08
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: > 
+ Current members of the National Assembly, National Council of Provinces,
+ and provincial legislatures and executives in South Africa.
+description: |
+    This dataset includes information about current members of South Africa's
+    National Assembly, National Council of Provinces, provincial legislatures,
+    and provincial executives. It covers various positions in the National
+    Legislature and Executive branch, including Ministers and members of the
+    Executive Committee.
+publisher: 
+  name: Parliamentary Monitoring Group (PMG)
+  description: |
+   The Parliamentary Monitoring Group (PMG) provides comprehensive details
+   about parliamentary activities and various stakeholders in South Africa.
+  url: https://pmg.org.za/
+  official: true
+  country: "za"
+url: https://pa.org.za/help/api
+data:
+  url: https://pa.org.za/media_root/popolo_json/pombola.json
+  format: JSON
+assertions:
+  min:
+    schema_entities:
+      Person: 450
```

Comment:
we expect at least 500 because of 400 MPs + 100 or more members of provincial legislatures